sys/vane/jael.hoon

```diff
@@ -84,7 +85,20 @@
   $:  source=(each ship node-src)                       ::  learning from
       heard=(set event-id)                              ::  processed events
       latest-block=@ud                                  ::  last heard block
+      foreign-block=@ud                                 ::  node's latest block
   ==                                                    ::
+++  state-snapshots                                     ::  rewind points
+  $:  interval=_100                                     ::  block interval
+      max-count=_10                                     ::  max snaps
```

Comment:
Maybe set these manually in `++init`? idk if value-by-default is considered good style in nu-hoon, but I rarely ever see this used. (Though, then, almost always the bunts are the desired defaults.)